samples/bookinfo/src/productpage/productpage.py

```diff
@@ -70,6 +71,13 @@
     "children": []
 }
 
+# Layotto Detail
+layottoDetails = {
+    "name": "http://127.0.0.1:8080",
```

Comment:
If we hardcode the url here, the demo can't run with istio
We should get this url using environment variables, like other services did:
![image](https://user-images.githubusercontent.com/26001097/179687291-e61cf5ee-33cb-4d23-bffd-a2edc5ebc551.png)